### DISCUSSION TITLE
[react-simple-maps] undocumented change of onMove event properties from k to zoom

### DISCUSSION THREAD

**Discussion by FunctionDJ** [DefinitelyTyped/DefinitelyTyped#62407] [Issues with a @types package] [answered]
In short, this line needs to be changed from "k" to "zoom" and the related test needs to be adapted. The test should fail with the latest version of react-simple-maps:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0c0777bd1dd9b34f4fa4e36aecfdc3228e64d454/types/react-simple-maps/index.d.ts#L81

Here's the commit that changed the property: https://github.com/zcreativelabs/react-simple-maps/commit/77e7678ebdf83e35fe05ca9020b65270e28d4617

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment, marked as answer]
I have created PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62495
